Makefile

```
@@ -18,139 +18,84 @@ IMAGE_LATEST_TAG=$(IMAGE_REGISTRY)/$(REGISTRY_ORG)/$(REGISTRY_REPO):latest
-	make create-oper
-	make create-service-and-db
-
-.PHONY: delete-all
-delete-all:
-	@echo Delete Mobile Security Service Operator, Service and namespace ${NAMESPACE}:
-	- kubectl delete -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityserviceapp_crd.yaml
-	- kubectl delete -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservice_crd.yaml
-	- kubectl delete -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservicedb_crd.yaml
-	- kubectl delete -f deploy/cluster_role.yaml
-	- kubectl delete -f deploy/cluster_role_binding.yaml
-	- kubectl delete -f deploy/service_account.yaml
-	- kubectl delete -f deploy/operator.yaml
-	- kubectl delete namespace ${NAMESPACE}
-
-.PHONY: create-oper
-create-oper:
-	@echo Create Mobile Security Service Operator:
+#INSTALL/UNINSTALL
+.PHONY: setup-project
+setup-project:
+	@echo ....... Logging in system:admin user ....... 
+	- oc login -u system:admin
+	@echo ....... Creating namespace ....... 
 	- oc new-project ${NAMESPACE}
-	- kubectl create -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservice_crd.yaml
-	- kubectl create -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservicedb_crd.yaml
-	- kubectl create -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityserviceapp_crd.yaml
-	- kubectl create -f deploy/cluster_role.yaml
+	- oc project ${NAMESPACE}
+	@echo ....... Adding Mobile Security Service CRDS ....... 
+	- kubectl create -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservice_crd.yaml 
+	- kubectl create -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityservicedb_crd.yaml 
+	- kubectl create -f deploy/crds/mobile-security-service_v1alpha1_mobilesecurityserviceapp_crd.yaml 
+	- kubectl create -f deploy/cluster_role.yaml 
 	- kubectl create -f deploy/cluster_role_binding.yaml
-	- kubectl create -f deploy/service_account.yaml
+	- kubectl create -f deploy/service_account.yaml 
+	- oc new-project ${APP_NAMESPACE}
+
+.PHONY: install-mss
+install-mss:
+	@echo ....... Logging in system:admin user ....... 
+	- oc login -u system:admin
+	@echo ....... Create Mobile Security Service Operator and Service in the namespace ${NAMESPACE} ....... 
 	- kubectl create -f deploy/operator.yaml
+	make install-service-and-db
+
+.PHONY: uninstall-mss
+uninstall-mss:
+	@echo ....... Create Mobile Security Service Operator and Service in the namespace ${NAMESPACE} ....... 
+	- kubectl delete -f deploy/operator.yaml
+	make uninstall-service-and-db
 
-.PHONY: delete-oper
-delete-oper:
-	@echo Deleting Mobile Security Service Operator and namespace:
+.PHONY: bind-app
+bind-app:
+	@echo ....... Creating an example app CR simulating MDC Bind ......
+	kubectl apply -f deploy/crds/examples/mobile-security-service_v1alpha1_mobilesecurityserviceapp_cr.yaml -n ${APP_NAMESPACE}
+	
+.PHONY: unbind-app
+unbind-app:
+	@echo ....... Deleting an example app CR simulating MDC Bind ......
```

We should not add any info Regards MDC. 
This project can be used by MDC as any other product. We cannot create ties one it. Also, it is an OpenSource project and could be used standalone. 

```suggestion
	@echo ....... Deleting the MobileSecurityServiceApp example CR from the current namespace  ......
```